ingestion/functions/parsing/brazil_paraiba/paraiba.py

```diff
@@ -0,0 +1,193 @@
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+if ('lambda' not in sys.argv[0]):
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+import parsing_lib
+
+_DATE_DEATH_INDEX = 6
+_GENDER_INDEX = 3
+_AGE_INDEX = 5
+_METHOD_CONFIRMATION_INDEX = 0
+_DATE_SYMPTOMS_INDEX = 2
+_MUNICIPALITY_INDEX = 1
+_PREEXISTING_CONDITIONS_INDEX = 4
+
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+
+    The date is listed in YYYY-mm-dd format, but the date filtering API
+    expects mm/dd/YYYYZ format.
+    """
+    date = datetime.strptime(raw_date, "%Y-%m-%d")
+    return date.strftime("%m/%d/%YZ")
+
+def convert_gender(raw_gender: str):
+    if raw_gender == "Masculino":
+        return "Male"
+    elif raw_gender == "Feminino":
+        return "Female"
+    return None
+
+def convert_age(age: str):
+    if age.isdecimal():
+        return {
+            "start": float(age),
+            "end": float(age)
+        }
+    else:
```

Comment:
Comment on what format is the age if not all decimal please, give an example.

Reply:
Do you mean in the script or here?

Reply:
in the script, for future readers of this parser (people who will want to understand how data was parsed won't look at this PR but at the code), thank you.

Reply:
nice one, done.